### PR TITLE
Build on aarch64

### PR DESCRIPTION
**Pull request by hfiguiere**
Build on arch64.

I don't have the hardware to test but the app start and the assistant works until I reach the router selection.
Tested on a Raspberry Pi with Raspberry Pi OS 64-bit.

### CODE REVIEW

**Comment by hfiguiere** [comment]
Build failure because the patch also need to be applied to x86_64. Will update the PR.

File: org.tabos.roger.json [hunk 1]
```diff
@@ -229,6 +233,12 @@
                     "url" : "https://gitlab.com/tabos/spandsp.git",
                     "tag" : "0.0.6pre21",
                     "commit" : "68455c2d0d56def0443ba2ae7df0e82e3a5fa5bb"
+                },
+                {
```
Comment (janbrummer) [inline org.tabos.roger.json:237]: I do not understand why this part is necessary, could you give more hints please?

**Comment by janbrummer** [comment]
Thanks a lot for your PR, i really appreciate it!

File: org.tabos.roger.json (hunk 1 above)
Comment (hfiguiere) [reply org.tabos.roger.json:237]: the `config.guess` and `config.sub` files are too old to detect `aarch64` and the build fail.

The thing that's weird here is that there are in git while usually these files are only disted in the tarball...

**Pull request merged**